lib/shared/global_configuration.rb

```diff
@@ -89,6 +90,14 @@ def freeze(key, value)
       @frozen_keys << (aliases[key] || key)
     end
 
+    def blank_value?( key )
+      self[key].blank? || self[key] == 'not set'
+    end
+
+    def present_value?( key )
+      !self[key].blank? && self[key] != 'not set'
```

Comment:
maybe it's easier to just call the blank_value? method and include a bang before it.